src/core/system.c

```diff
@@ -105,20 +110,46 @@ void Sys_ResetTempMem()
 SYS TIME
 ===============================================================================
 */
-float Sys_FloatTime (void)
+
+#if _MSC_VER
+typedef struct timeval {
+	long tv_sec;
+	long tv_usec;
+} timeval;
+
+int gettimeofday(struct timeval * tp, struct timezone * tzp)
```

Comment:
I _think_ SDL_GetPerformanceFrequency may change when the processor changes its frequency.